GCMtools/gcm_plotting.py

```diff
@@ -0,0 +1,176 @@
+        the units specified in the dataset attributes (e.g., init of GCMT object).
+    time : int, optional
+        Timestamp that should be plotted. By default, the last time is
+        selected.
+    lookup_method : str, optional
+        The look-up method that is used to slice along pressure:
+        'exact' for exactly matching key look-up (default);
+        'nearest' to pick out the nearest neighbour Z coordinate;
+        'interpolate' for a linear interpolation along the Z axis.
+    ax : matplotlib.axes.Axes, optional
+        The axis on which you want your plot to appear.
+    plot_windvectors : boolean, optional
+        If True (default), plot the horizontal wind vectors over the colour
+        mesh.
+    wind_kwargs : dict, optional
+        Additional keywords for the method plot_horizontal_wind.
+    cbar_kwargs : dict, optional
+        Additional keywords for the colorbar.
+    fs_labels : int, optional
+        Optionally set font size of the axis labels.
+    fs_ticks : int, optional
+        Optionally set font size of the tick labels.
+    title : str, optional
+        Title for the isobaric slice plot. By default, the selected pressure
+        and time stamp of the slice are displayed.
+    xlabel : str, optional
+        X-axis label, longitude by default.
+    ylabel : str, optional
+        Y-axis label, latitude by default.
+    """
+    if ax is None:
+        fig= plt.figure()
+        ax = plt.gca()
+    if wind_kwargs is None:
+        wind_kwargs = {}
+    if cbar_kwargs is None:
+        cbar_kwargs = {}
+
+    # retrieve default units
+    p_unit = ds.attrs.get('p_unit')
+    time_unit = ds.attrs.get('time_unit')
+
+    # if no timestamp is given, pick the last available time
+    if time == -1:
+        time = ds.time.isel(time=-1).values
+    # time-slice of the dataset
+    # (note: the look-up method for time is always assumed to be exact)
+    this_time = ds.time.sel(time=time).values
+    ds = ds.sel(time=time)
+
+    # isobaric slice based on the look-up method for pressure
+    if lookup_method == 'exact':
+        this_p = ds.Z.sel(Z=p).values
+        ds2d = ds.sel(Z=p)
+    elif lookup_method == 'nearest':
+        this_p = ds.Z.sel(Z=p, method="nearest").values
+        ds2d = ds.sel(Z=p, method='nearest')
+    elif lookup_method == 'interpolate':
+        this_p = ds.Z.interp(Z=p).values
+        ds2d = ds.interp(Z=p)
+    else:
+        raise ValueError("Please enter 'exact', 'nearest', or 'interpolate' as Z lookup method.")
+
+    # Simple plot (with xarray.plot.pcolormesh)
+    plotted = ds2d[var_key].plot(add_colorbar=False, **kwargs)
+
+    # make own colorbar, as the automatic colorbar is hard to customize
+    cbar = plt.colorbar(plotted, ax=ax, **cbar_kwargs)
+    if 'label' in cbar_kwargs:  # look if a colorbar label is given
+        cbar_label = cbar_kwargs['label']
+    else:
+        cbar_label = var_key    # if not, use label of the plotted quantity
```

Comment:
```Suggestion
    cbar_label = cbar_kwargs.get('label', var_key)
```

Reply:
A littlebit simpler :D

Reply:
Neat! I will use this for dictionaries in the future.